src/main/java/mate/academy/spring/controller/AuthenticationController.java

```diff
@@ -0,0 +1,28 @@
+package mate.academy.spring.controller;
+
+import mate.academy.spring.mapper.DtoResponseMapper;
+import mate.academy.spring.model.User;
+import mate.academy.spring.model.dto.request.UserRequestDto;
+import mate.academy.spring.model.dto.response.UserResponseDto;
+import mate.academy.spring.security.AuthenticationService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AuthenticationController {
+    private final AuthenticationService authenticationService;
+    private final DtoResponseMapper<UserResponseDto, User> responseMapper;
+
+    public AuthenticationController(AuthenticationService authenticationService,
+                                    DtoResponseMapper<UserResponseDto, User> responseMapper) {
+        this.authenticationService = authenticationService;
+        this.responseMapper = responseMapper;
+    }
+
+    @PostMapping("/register")
+    public UserResponseDto registration(@RequestBody UserRequestDto requestDto) {
+        User user = authenticationService.register(requestDto.getEmail(), requestDto.getPassword());
+        return responseMapper.toDto(user);
```

Comment:
Why is this method not void? Even in the example:
![image](https://user-images.githubusercontent.com/93381314/207717282-1fc33998-98f6-4a40-96f1-bd6662d11b70.png)

Reply:
I'm not sure why, maybe to provide info that registration was successful, for example to parse username from response and display it in success message. Also our register method returning value for some reason, and in the next HW's register also not void

Reply:
Okay, let's leave this question to the mentors